debian/rules

```diff
@@ -155,18 +154,28 @@ binary-arch: build install
 	sed -i s./etc/sysconfig/./etc/default/.g debian/ceph-base/lib/systemd/system/ceph-create-keys@.service
 	sed -i s./etc/sysconfig/./etc/default/.g debian/ceph-osd/lib/systemd/system/ceph-osd@.service
 	sed -i s./etc/sysconfig/./etc/default/.g debian/ceph-osd/lib/systemd/system/ceph-disk@.service
+	install -m0644 systemd/ceph-mon.target debian/ceph-mon/lib/systemd/system
+	install -m0644 systemd/ceph-osd.target debian/ceph-osd/lib/systemd/system
 
 	install -d -m0755 debian/ceph-mds/lib/systemd/system
 	install -m0644 systemd/ceph-mds@.service debian/ceph-mds/lib/systemd/system
 	sed -i s./etc/sysconfig/./etc/default/.g debian/ceph-mds/lib/systemd/system/ceph-mds@.service
+	install -m0644 systemd/ceph-mds.target debian/ceph-mds/lib/systemd/system
 
 	install -d -m0755 debian/radosgw/lib/systemd/system
 	install -m0644 systemd/ceph-radosgw@.service debian/radosgw/lib/systemd/system
 	sed -i s./etc/sysconfig/./etc/default/.g debian/radosgw/lib/systemd/system/ceph-radosgw@.service
+	install -m0644 systemd/ceph-radosgw.target debian/radosgw/lib/systemd/system
 
 	install -d -m0755 debian/rbd-mirror/lib/systemd/system
 	install -m0644 systemd/ceph-rbd-mirror@.service debian/rbd-mirror/lib/systemd/system
 	sed -i s./etc/sysconfig/./etc/default/.g debian/rbd-mirror/lib/systemd/system/ceph-rbd-mirror@.service
+	install -m0644 systemd/ceph-rbd-mirror.target debian/rbd-mirror/lib/systemd/system
+
+	dh_systemd_enable
+	dh_installinit -p ceph-base --name ceph --no-start
+	dh_installinit -p radosgw --no-start
+	dh_systemd_start --restart-after-upgrade
```

Comment:
> --restart-after-upgrade
>           Do not stop the unit file until after the package upgrade has been completed.

this reminds me how we do the upgrade

```
   sequential:
   - install.upgrade:
       mon.a:
       mon.b:
   - ceph.restart: [mon.a, mon.b, mon.c, mds.a, osd.0, osd.1, osd.2, osd.3]
```

so i thought we didn't want to stop the service until upgrade is finished. but yeah. i can drop that option.